internal/resources/resource_load_balancer.go

```diff
@@ -75,17 +75,19 @@ func LoadBalancer() *schema.Resource {
 						"size": {
 							Type:        schema.TypeString,
 							Optional:    true,
+							Default:     "SMALL",
 							Description: `In Filter. Supported Values are "SMALL", "MEDIUM", "LARGE"`,
 						},
-						"loglevel": {
+						"log_level": {
 							Type:        schema.TypeString,
 							Optional:    true,
+							Default:     "INFO",
 							Description: `In Filter. Supported Values are "DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL", "ALERT", "EMERGENCY"`,
 						},
 						"tier1": {
 							Type:        schema.TypeString,
 							Optional:    true,
-							Default:     true,
+							Default:     "/infra/tier-1s/26cdb82e-0057-4461-ad4d-cddd61d77b1f",
```

Comment:
Will this be static for all the environments?

Reply:
yes

Reply:
currently we have tier1 supporting for creating loadbalancer and its default one